lib/controllers/issueish-list-controller.js

```diff
@@ -90,15 +90,14 @@ export class BareIssueishListController extends React.Component {
     return null;
   }
 
-  openOnGitHub = url => {
-    return new Promise((resolve, reject) => {
-      shell.openExternal(url, {}, err => {
-        if (err) { reject(err); } else {
-          resolve();
-          addEvent('open-issueish-in-browser', {package: 'github', component: this.constructor.name});
-        }
-      });
-    });
+  openOnGitHub = async url => {
+    try {
+      const resolvedPromise = await shell.openExternal(url);
+      addEvent('open-issueish-in-browser', {package: 'github', component: this.constructor.name});
+      return resolvedPromise;
+    } catch (err) {
+      throw new Error(err);
+    }
```

Comment:
We can simplify these!

The try/catch statements aren't actually doing anything, since they're just re-throwing any error that's raised. We also don't need to return `resolvedPromise` since the return value isn't significant. So these functions can look like this:

```suggestion
    await shell.openExternal(url);
    addEvent('open-issueish-in-browser', {package: 'github', component: this.constructor.name});
```

Reply:
Hey @smashwilson. Good point. Done 👍